docker-compose/.env

```diff
@@ -22,5 +22,5 @@ MSSQL_DB_PORT=1433
 
 #sqlite | postgres | mssql
 DATABASE_ENGINE=postgres
-VERSION_TAG=0.6.0
+VERSION_TAG=0.7.0
```

Comment:
if this update is done before the release, then it might happen that someone using docker compose will face issues running it, as the images are not available yet